stl/inc/ranges

```diff
@@ -118,19 +118,19 @@ namespace ranges {
         _Pipeline(_Ty1, _Ty2) -> _Pipeline<_Ty1, _Ty2>;
 
         template <class _Left, class _Right>
-            requires _Range_adaptor_closure_object<_Left> //
-                && _Range_adaptor_closure_object<_Right> //
-                && constructible_from<remove_cvref_t<_Left>, _Left> //
-                && constructible_from<remove_cvref_t<_Right>, _Right>
-        constexpr auto operator|(_Left&& __l, _Right&& __r) noexcept(
-            noexcept(_Pipeline{static_cast<_Left&&>(__l), static_cast<_Right&&>(__r)})) {
+        requires _Range_adaptor_closure_object<_Left> //
+            && _Range_adaptor_closure_object<_Right> //
+            && constructible_from<remove_cvref_t<_Left>, _Left> //
+            && constructible_from<remove_cvref_t<_Right>, _Right> //
+                _NODISCARD constexpr auto operator|(_Left&& __l, _Right&& __r) noexcept(
+                    noexcept(_Pipeline{static_cast<_Left&&>(__l), static_cast<_Right&&>(__r)})) {

```

Comment:
Ugh, this formatting is hideous, but not worth resetting the port. I'll clean these in a followup.

Reply:
Thanks. Yeah, I noticed how icky it was - perhaps because `[[nodiscard]]` is being concealed with the macro.